### DISCUSSION TITLE
Samba共享文件夹上传单文件超过4MB时提示空间不足

### DISCUSSION THREAD

**Discussion by wqianglxf** [istoreos/istoreos#769] [Q&A] [answered]
Samba共享文件夹复制大文件时提示空间不足
![74df19a1dac19092d94a3d7c9331b2da](https://user-images.githubusercontent.com/66858128/236659166-b7b8a6b3-a0f6-4242-86b9-3a9aa2e69fe6.png)
可以确定是有空间的，够存储使用
![5cb994c5af70e32a648b3021afadcfcd](https://user-images.githubusercontent.com/66858128/236659169-7df595fe-e238-4b4c-bd38-737496aa018d.png)
尝试了4G及以下的文件，可以正常复制
![d1f36a74501ce898ae44af482848f1c4](https://user-images.githubusercontent.com/66858128/236659173-c6b21a13-fd29-41b4-b658-ca39eaf54722.png)
超过4G的无法复制

**Comment by jjm2473** [comment, marked as answer]
不要共享/mnt，而是共享/mnt下的挂载点，不然windows会以为你共享的空间只有4MB
